e2e/registration_test.go

```diff
@@ -73,7 +131,8 @@ func TestE2E_ServiceRegistration_DeregisterWhenStopped(t *testing.T) {
 	id := "cts-01"
 	configPath := filepath.Join(tempDir, configFile)
 	config := baseConfig(tempDir).appendID(id).
-		appendConsulBlock(srv).appendTerraformBlock().appendDBTask()
+		appendConsulBlock(srv).appendTerraformBlock().
+		appendModuleTask("disabled_task", "mkam/hello/cts", "enabled = false")
```

Comment:
Why does the task need to be disabled?

Reply:
It doesn't, this was an optimization since whether the task runs or not doesn't affect the test case.

Reply:
Added a comment with the same information!